core/lib/config/src/configs/api.rs

```diff
@@ -213,6 +213,13 @@ pub struct Web3JsonRpcConfig {
     /// (additionally to natively bridged tokens).
     #[serde(default)]
     pub whitelisted_tokens_for_aa: Vec<Address>,
+    /// Enabled JSON RPC API namespaces. If not set, all namespaces will be available
+    #[serde(default)]
+    pub api_namespaces: Option<Vec<String>>,
```

Comment:
IMO, we can afford to be more typesafe here, i.e. declare a namespace enum (or move `Namespace` from the API server crate) and use `Option<HashSet<Namespace>>`. Or do you think we should allow for custom namespaces be defined by the node?

Reply:
I was trying to do it. But it doesn't seem like a proper solution, it will be typed safety a bit later. I'd prefer to keep it string here and convert later

Reply:
Agree, but i really don't want to move namespaces to the places where are they are not belong it's part of the api

Reply:
Right. As discussed previously, we could decentralize configs (so that API server defines its config in its crate), but of course this would require reworking config approach entirely.